Classes/Core/Functional/Framework/DataHandling/DataSet.php

```diff
@@ -70,13 +71,23 @@ public static function import(string $path): void
         $dataSet = self::read($path, true);
         foreach ($dataSet->getTableNames() as $tableName) {
             $connection = GeneralUtility::makeInstance(ConnectionPool::class)->getConnectionByName(ConnectionPool::DEFAULT_CONNECTION_NAME);
+            $platform = $connection->getDatabasePlatform();
```

Comment:
I think we can move the two lines out of the outer foreach()

Reply:
Done